app/controllers/items_controller.rb

```diff
@@ -0,0 +1,62 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  def index
+    @items = Item.all
+  end
+
+  def show; end
+
+  def new
+    @item = Item.new
+    @group = Group.find(params[:group_id])
+  end
+
+  def edit; end
+
+  def create
+    @item = Item.new(item_params)
+    @item.user = current_user
+    @group = Group.find(params[:group_id])
+    respond_to do |format|
+      if @item.save
+        format.html { redirect_to group_path(@group), notice: 'Item was successfully created.' }
+        format.json { render :show, status: :created, location: @item }
+      else
```

Comment:
[ OPTIONAL ]

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better if you make the home page as in the Design that has two buttons (sign up and login )
- Home page design 
<img width="236" alt="Screenshot 2023-02-15 at 15 26 59" src="https://user-images.githubusercontent.com/61790587/219056997-d2289806-57fe-4113-969c-155dd2103b20.png">